### ISSUE TITLE
import error on Nautilus's "move to other AppVM"

### ISSUE THREAD

**Issue by Scinawa** [QubesOS/qubes-issues#1803]
Hi my fellows qubers!

It happened a few times that trying to move a file from a Debian8 appvm (by right-clicking the file and pressing "move file to") results in no action at all.
Calling nautilus from command lines shows that:

```
Traceback (most recent call last):
  File "/usr/bin/qvm-mru-entry", line 24, in <module>
    import gtk
ImportError: No module named gtk
```

And so, this error can be simply fixed by typing:

`sudo apt install python-gtk2`

I'm on qubes 3 rc3 testing.

xoxo everyone!

**Comment by marmarek** [comment]
What version of qubes-core-agent package do you have? This is already
fixed some time ago (qubes-core-agent >= 3.1.10)

Are you updating with `apt-get dist-upgrade`? Just `apt-get upgrade` do
not pull new packages (which is needed in this case).

-- 
Best Regards,
Marek Marczykowski-Górecki
Invisible Things Lab
A: Because it messes up the order in which people normally read text.
Q: Why is top-posting such a bad thing?

**Comment by marmarek** [comment]
Duplicate of #1567
